### ISSUE TITLE
Saw non-null label

### ISSUE THREAD

**Issue by kei6** [yu20103983/FOTS#6]
Running command: `python multi_gputrain.py`
Error: `Saw non-null label`

I've fixed this error by change `self.NUM_CLASSES` into `self.NUM_CLASSES+1` in `recognizer.py`, because CTC loss in tensorflow need number of class plus 1 to work with:
```
self.logits = slim.fully_connected(lstmout, self.NUM_CLASSES+1, activation_fn=None, scope='fc')
```
This action help to pass the bug.
I'm on my work to check its right for training.
